public/app/features/dashboard/components/AddPanelWidget/AddPanelWidget.tsx

```diff
@@ -119,7 +120,7 @@ export const AddPanelWidgetUnconnected: React.FC<Props> = ({ panel, dashboard, u
     const newPanel: PanelModel = {
       ...panelInfo.model,
       gridPos,
-      libraryPanel: _.pick(panelInfo, 'name', 'uid', 'meta'),
+      libraryPanel: toPanelModelLibraryPanel(panelInfo),
```

Comment:
I missed this in #31590 so any new library panels aren't updated correctly without this fix.